manifests/bootkube/manifests/kube-apiserver-daemonset.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         k8s-app: "kube-apiserver"
         openshift.io/control-plane: "true"
         openshift.io/component: "api"
+        checkpointer.alpha.coreos.com/checkpoint: "true"
```

Comment:
do we want to checkpoint these? i thought we want to checkpoint only operator created resources

/cc @deads2k

Reply:
Makes sense. removing.